src/Server.cpp

```diff
@@ -51,8 +51,7 @@ void Server::startPollEventLoop()
 			{
 				if (_FDs[i].revents & (POLLIN | POLLOUT))
 				{
-					std::cout << "i: " << i << std::endl;
-					std::cout << "Enters revents" << std::endl;
+					Debug::log("Enters revents", Debug::OCF);
```

Comment:
The key OCF is to print Orthodox Canonical Statement, which is not the case here. A new meaningful key (is an enum I don't remember should be created here). Maybe SERVER or POLL, to print statements related to the poll flow. This is not relevant for this issue so you can eventually open an extra issue for that! But happy that you used it! 🙌

Reply:
I did not understand what is OCF at all, I thought something different. Will fix in the next PR!